### PR TITLE
Oc/mcka master merge

### PR DESCRIPTION
**Pull request by Kelketek**
Merge with the recent changes to the mcka fork.

@itsjeyd If you could look this over again, same method as before, just need to make sure pulling the changes back into our fork didn't cause a regression.

### CODE REVIEW

File: poll/public/html/poll.html [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
-    <div class="poll-feedback-container{% if not choice %} poll-hidden{% endif %}">
-        <hr />
-        <h3 class="poll-header">Feedback</h3>
-        <div class="poll-feedback">
-            {{feedback|safe}}
-        </div>
-    </div>
-    {% endif %}
+    <div class="poll-block-form-wrapper">
+      <h3 class="poll-header">{{display_name}}</h3>
```
Comment (itsjeyd) [inline poll/public/html/poll.html]: @Kelketek Indentation is inconsistent here; you're using 4 spaces below, but this line is only indented by two spaces.

File: poll/public/html/poll.html [hunk 2]
```diff
@@ -1,42 +1,46 @@
...
+      <form>
+          <div class="poll-question-container">
+              {{question|safe}}
+          </div>
+          <ul class="poll-answers">
+          {% for key, value in answers %}
+              <li class="poll-answer">
+                  <div class="poll-input-container">
+                      <input type="radio" name="choice" id="{{url_name}}-answer-{{key}}" value="{{key}}" {% if choice == key %}checked{% endif %}/>
```
Comment (itsjeyd) [inline poll/public/html/poll.html]: @Kelketek Nit: You might want to reformat this to span multiple lines; it's a bit hard to read right now. Same goes for L28 and L30 below.

**Comment by itsjeyd** [comment]
@Kelketek There's a small glitch that concerns the bars that are displayed after submitting an answer to a poll question:

poll-answer-bar-too-small

For the first item, the bar that shows the percentage is not as tall as the background. This isn't the case for the other items. I don't remember running into this the first time I tested. It also affects Apros. Not sure if this is really an issue, but wanted to point it out anyway.

**Comment by Kelketek** [comment]
@itsjeyd This is not a bug. The old version of this code only highlighted the text, and this does the same, only the text can be on multiple lines. See https://raw.githubusercontent.com/open-craft/xblock-poll/master/doc_img/poll_mixed_result.png

HTML reformatted.

**Comment by itsjeyd** [comment]
@Kelketek 

> This is not a bug.

OK, never mind then :)

:+1: once the tests pass.

**Pull request merged**